projects/uitdatabank/toc.json

```diff
@@ -50,7 +50,15 @@
     {
       "type": "item",
       "title": "Entry API",
-      "uri": "reference/entry.json"
+      "uri": "reference/entry.json",
+      "item": [
```

Comment:
This doesn't work, you cannot add a menu item underneath a reference to an OpenAPI file. You'll need to add a new "Entry API" group under the "Guides" divider instead. See the Search API guides as an example: https://github.com/cultuurnet/apidocs/pull/33/files#diff-b3bfd55442c8e8f3eb459d3dc57b189d68dfc36317d2f49158091767e680cc68R22-R57

Reply:
Ah, yes. Fixed it in a new commit